src/Components/SceneList/Internal/SceneDialog.tsx

```diff
@@ -103,24 +123,12 @@ const SceneDialog: React.FC<ISceneDialogProps> = ({
     onEditScene,
     renderBlobDropdown
 }) => {
-    const [newSceneName, setNewSceneName] = useState('');
-    const [newSceneDescription, setNewSceneDescription] = useState('');
-    const [newLatitudeValue, setNewLatitudeValue] = useState(undefined);
-    const [newLongtitudeValue, setNewLongtitudeValue] = useState(undefined);
-    const [newSceneBlobUrl, setNewSceneBlobUrl] = useState('');
+    const [state, dispatch] = useReducer(
+        SceneDialogReducer,
```

Comment:
shouldn't we use the sceneToEdit to populate the reducer initial state? We shouldn't need to fork all the update functions based on edit/create. They should all be operating on the same state object, that's the point of hte reducer.